psa-crypto/tests/mod.rs

```diff
@@ -0,0 +1,113 @@
+// Copyright 2020 Contributors to the Parsec project.
+// SPDX-License-Identifier: Apache-2.0
+
+use psa_crypto::types::algorithm::{Algorithm, AsymmetricSignature, Hash};
+use psa_crypto::types::key::{Attributes, Lifetime, Policy, Type, UsageFlags};
+
+#[test]
+fn generate_integration_test() {
```

Comment:
Nice, thanks for the new integration tests!
Nice to see that the CI passed them as well!